web/src/compose.js

```diff
@@ -706,6 +734,27 @@ export function initialize() {
         }
     });
 
+    $("body").on("click", ".audio_link", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        let $target_textarea;
+        let edit_message_id;
+        if ($(e.target).parents(".message_edit_form").length === 1) {
+            edit_message_id = rows.id($(e.target).parents(".message_row"));
+            $target_textarea = $(`#edit_form_${CSS.escape(edit_message_id)} .message_edit_content`);
+        }
+
+        const show_audio_chat_button = compute_show_audio_chat_button();
+        if (!show_audio_chat_button) {
+            return;
+        }
```

Comment:
Is this conditional necessary? It seems like we should be able to assume that's the case because the button was there to be clicked.

Reply:
This is kind of true. But this disallows the user to do a `querySelector` for the video call button and then simulate the click from the console.

Reply:
OK -- that doesn't seem important as a thing to prevent throwing an exception?

Anyway, it's probably fine to have this in case we fail to live-update the button state somehow, but it seems a bit silly.